src/NodaTime/LocalDate.cs

```diff
@@ -140,7 +140,9 @@ public LocalDate(Era era, int yearOfEra, int month, int day, CalendarSystem cale
 
         /// <summary>Gets the calendar system associated with this local date.</summary>
         /// <value>The calendar system associated with this local date.</value>
-        public CalendarSystem Calendar => CalendarSystem.ForOrdinal(yearMonthDayCalendar.CalendarOrdinal);
+        public CalendarSystem Calendar => CalendarSystem.ForOrdinal(CalendarOrdinal);
+
+        private CalendarOrdinal CalendarOrdinal => yearMonthDayCalendar.CalendarOrdinal;
```

Comment:
(I wondered if this maybe could do with a comment, though initially I misread this as a field rather than a property.)

Reply:
I think it's probably okay as it is. The comment would probably be a pretty no-op one. (And it's private, so it's not like we need to worry about code in other types getting confused.)